content/docs/1.3.0/tools/iofogctl/translating.md

```diff
@@ -0,0 +1,94 @@
+### Translating your yml files to the 1.3.0 standard
+
+There is a couple quick changes you can do to your current yml file to make them 1.3.0 compatible if you are using the 1.2.x standard.
+
+We now follow a more k8s style formatting by using apiVersions, kinds, metadata and spec top level keys.
+
+All of our current top level keys in 1.2.x will now be handled under spec.
+
+Previous Spec:
+
+```yaml
+---
+controllers:
+  - name: LocalController
+    host: localhost
+    iofoguser:
+      name: Quick
+      surname: Start
+      email: user@domain.com
+      password: q1u45ic9kst563art
+
+agents:
+  - name: LocalAgent
+    host: localhost
+```
+
+Current Spec:
+
+```yaml
+---
+apiVersion: iofog.org/v1
+kind: ControlPlane
+metadata:
+  name: ymlspec
+  namespace: default
+spec:
+  iofogUser:
+    name: iofog
+    surname: user
+    email: host@domain.com
+    password: mysecretpw
+  controllers:
+    - name: controller
+      user: default
+      host: xxx.xxx.x.xxx #(ip address)
+      keyFile: ~/.ssh/id_rsa
+---
+apiVersion: iofog.org/v1
+kind: Connector
+metadata:
+  name: metdata
+  namespace: default
+spec:
+  user: connector
+  host: xxx.xxx.x.xxx #(ip address)
+  keyfile: ~/.ssh/id_rsa
+---
+apiVersion: iofog.org/v1
+kind: Agent
+metadata:
+  name: agent-name
+  namespace: default
+spec:
+  user: agent
+  host: xxx.xxx.x.xxx #(ip address)
+  keyFile: ~/.ssh/id_rsa
+```
+
+Note: we seperate kinds by three dashes, this allows you to use a single file, to deploy all your infrastructure and keep it legible.
+
+As you can see, the top keys are now standard for whatever you want to deploy, which you simply identify in the `kind` key
+the name and namespaces you were operating in are now handled under the metadata tag, and the spec tag, while different for each kind, contains
+the same kind of data as the previous spec, simply under spec. apiVersion is standard for now, but will be updated to reflect changed in spec for
```

Comment:
apiVersion sentence is confusing. Just state that our latest YAML spec version is 1 but this may be updated in the future.